trino/sqlalchemy/dialect.py

```diff
@@ -55,6 +55,9 @@ class TrinoDialect(DefaultDialect):
     supports_multivalues_insert = True
     postfetch_lastrowid = False
 
+    # Caching
```

Comment:
can you add a comment saying why it's disabled instead?

Something like `warnings are generated if not explicitly set and needs tests before it can be enabled`?